### DISCUSSION TITLE
Make Q-Input work like Stackoverflow Tags input

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#10944] [General - Components / Directives / etc] [answered]
Does anyone know of a way to make the Q-Input work similar to Stackoverflow's Tag input. Here is what im trying to do

When a user hit's comma on the keyboard it automatically appends the 'latest text' as a Chip into the input rather than using the the Enter key which QInput currently uses.

![image](https://user-images.githubusercontent.com/1416749/136701409-563630e3-781f-4aa4-99a2-38666f284ce9.png)

My suggestion was to just capture the @keyUp event and when the user presses the comma key, i could simply just mimc whatever the 'enter' command does. But im not sure how to do that.

**Comment by rstoenescu** [comment, marked as answer]
You need a QSelect. Look at the examples in https://quasar.dev/vue-components/select#create-new-values
